org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -62,6 +62,14 @@ <h3>Non-functional Changes</h3>
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/910">#910</a>).</li>
 </ul>
 
+<h3>API Changes</h3>
+<ul>
+    <li>The coverage check API and tools (Ant, Maven) now report an error, when
+        a coverage ratio limit is configured outside the range [0,1] to avoid
+        common configuration mistakes.
```

Comment:
There is dot is after reference to ticket as like in all existing places, so this one should be removed for consistency.